manifests/crc/BYODB-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -76,6 +76,18 @@ spec:
         app: kruize
         name: kruize
     spec:
+      initContainers:
+        - name: wait-for-postgres
+          image: quay.io/kruizehub/postgres:15.2 # Use the same Postgres version as the Postgres deployment
+          imagePullPolicy: IfNotPresent
+          command:
+            - sh
+            - -c
+            - |
+              until pg_isready -h postgres-service -p 5432 -U admin; do
```

Comment:
Is it tested on openshift .. Does it need execute permission ?

Reply:
Yes, this is tested on openshift, have added the logs for the same above.